data.json

```
@@ -682,5 +682,11 @@
     "githubUsername": "SubhamRaoniar28",
     "imageUrl": "https://user-images.githubusercontent.com/41485604/111068853-767f3e00-84f0-11eb-9122-80af0d842463.jpg",
     "issueNumber": 397
+  },
+  {
+    "name": "Pranjal Verma",
+    "githubUsername": "pvcodes",
+    "imageUrl": "https://avatars3.githubusercontent.com/pvcodes",
```

This shouldn't be your user avatar it should be an image of your profile
```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/54075838/111760905-40153a80-88c5-11eb-9600-aa49a78ddd05.png",
```